ruby_review_week_four.rb

```diff
@@ -26,12 +47,37 @@
 
 # b. on a second line, give that fish object’s name attribute the value of ‘Fred’ (assume you have access).
 
+
+
 # 12. Define a hash called ‘student’. This hash should have two keys, one called name, and the other called age, and assign a string to the name key, and an integer to the age key. 
+# student = {
+#   name: "Samaa",
+#   age: 30,
+# }
 
 # 13. Write a Ruby method that takes three arguments and returns the sum (total) of those three arguments.
 
+# def calculator(num1, num2, num3)
+#  puts num1 + num2 + num3
+# end
+
+# calculator(3,1,2)
+
 # 14. Write a Ruby method that takes an array and a string as arguments, and returns true if the string is in the array, and false if it is not.
 
+# arr =["apple", "banana", "grapes"]
+
+# def checker(arr,string)
+#   valid_answer = arr.include?(string)
+#   if valid_answer == true
```

Comment:
this is a really solid answer, the way you used `.include?` is perfect, careful again with the use of `puts`, make sure you `return` instead